### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_Summary_Keyboard:Focus is not visible on the copy and close button.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9298]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS2.4.7;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS2.4.7

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Select Configuration profile --> Update Azure settings --> Update Cluster settings --> Update Service settings & hit Next.
8. Navigate to the copy and close button.

**Actual:**
On navigating using tab to the close and copy button the focus is not visible on them.

**Expected:**
The focus should be clearly visible through black dotted lines on the button.

**User Impact:**
Keyboard users will face difficulty in knowing where the current focus is.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/ESOsZ2IsIZNEl1mMDmPxJtoBlUqp8d0or6a1mzspgsKpKg?e=2tybTe

**Comment by alanrenmsft** [comment]
dupe of #9321

**Comment by sspatil123** [comment]
As "SQL Server Big Data Cluster" feature has been removed.
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
